extensions/SwiftCloudFiles/php-cloudfiles-1.7.10/cloudfiles_http.php

```diff
@@ -1483,8 +1483,9 @@ private function _send_request($conn_type, $url_path, $hdrs=NULL, $method="GET",
 		while( $retriesLeft >= 0 ) {
 			$res = $this->_do_send_request( $conn_type, $url_path, $hdrs, $method, $force_new );
 
-			if ( !in_array( $res, [ 500, 503, false ] ) ) {
-				// request was successful, return
+			# PLATFORM-1059 - retry all HTTP 50x responses
+			if ( $res !== false && $res < 500 ) {
+				// request was successful (i.e. DFS backend did not fail), return
```

Comment:
Yeah, not it slightly less misleading. :-) I would actually remove it and leave "Retry on server errors (HTTP 50x responses)." :8ball: